### DISCUSSION TITLE
Device naming & battery information

### DISCUSSION THREAD

**Discussion by Floppe** [Flo-Schilli/homeassistant-grohe_sense#50] [Ideas]
Hi!

Would be nice if information could be added from which room they are installed. My own fault when I have left the default name for all devices so I have 3 identical.

I also miss the battery information.

// Floppe

**Comment by Flo-Schilli** [comment]
Hi Floppe,

Battery information is included in the newest release from today.
I'll check for room information during setup, so that you can easily assign it to rooms in HA, if that would be okay?

Regards 
Flo

**Comment by Flo-Schilli** [comment]
Hey Floppe,

I just added it in release this release: [v0.22.0-b3](https://github.com/Flo-Schilli/homeassistant-grohe_sense/releases/tag/v0.22.0-b3) maybe you can give it a try :smile:
